sample/src/main/java/com/karumi/dexter/sample/SampleActivity.java

```diff
@@ -144,11 +145,6 @@ private void createPermissionListeners() {
                 R.string.all_permissions_denied_feedback)
                 .withOpenSettingsButton(R.string.permission_rationale_settings_button_text)
                 .build());
-    cameraPermissionListener = new CompositePermissionListener(feedbackViewPermissionListener,
-        SnackbarOnDeniedPermissionListener.Builder.with(rootView,
-            R.string.camera_permission_denied_feedback)
-            .withOpenSettingsButton(R.string.permission_rationale_settings_button_text)
-            .build());
```

Comment:
@pedrovgs did you miss my last comment? I just need to accomplish this issue to finish the PR review. Thanks.